magic_spoiler/__main__.py

```diff
@@ -526,10 +526,13 @@ def get_spoiler_sets() -> List[Dict[str, str]]:
         return []
 
     spoiler_sets = []
+    # Find list of possible Set Types to exclude here: https://scryfall.com/docs/api/sets
+    excluded_set_types = ["alchemy", "masterpiece", "arsenal", "from_the_vault", "spellbook", "premium_deck", "duel_deck", "treasure_chest", "planechase", "archenemy", "vanguard", "box", "promo", "token", "memorabilia", "minigame"]
+
     for sf_set in sf_sets["data"]:
         if (
             sf_set["released_at"] >= time.strftime("%Y-%m-%d %H:%M:%S")
-            and sf_set["set_type"] != "token"
+            and sf_set["set_type"].lower() not in excluded_set_types
```

Comment:
aren't they all lowercase already?

Reply:
They are currently, yes.
Set types to exclude are defined in lower case too, which we control.

I thought it's good practice to ensure it's not breaking if scryfall might change. Not sure if that's needed or should not be done for some reason.

What do you think?

Reply:
it's not necessary, it leads contributors to think that this value is in a different casing while it isn't